tool/tctl/common/auth_command.go

```diff
@@ -221,6 +221,7 @@ var allowedCertificateTypes = []string{
 	"saml-idp",
 	"github",
 	"awsra",
+	"bound_keypair",
```

Comment:
should this be consistent with other types above using `-` instead of `_`?

what's the format being exported? seem current only TLS and SSH keys are exported in ca_export.go.

Reply:
Huh, I figured this list was used for `tctl auth rotate`, but apparently it only applies to `export`. We don't seem to export JWT CAs so I'll just remove it from the list entirely.